src/Microsoft.PowerShell.Commands.Management/commands/management/Computer.cs

```diff
@@ -1535,7 +1535,7 @@ protected override void BeginProcessing()
                             {
                                 string message = StringUtil.Format(ComputerResources.NoResorePoint, id);
                                 ArgumentException e = new ArgumentException(message);
-                                ErrorRecord errorrecord = new ErrorRecord(e, "NoResorePoint", ErrorCategory.InvalidArgument, null);
+                                ErrorRecord errorrecord = new ErrorRecord(e, "NoRestorePoint", ErrorCategory.InvalidArgument, null);
```

Comment:
Second argument in ErrorRecord ctor is [FullyQuilifiedErrorId](FullyQuilifiedErrorId), which is different from other cases in this PR, because it's technically a public API. However, we may want to change these misspelling. Can you separate these `FullyQuilifiedErrorId` changes in it's own PR?